internal/pkg/metadb/metadb.go

```diff
@@ -938,22 +946,6 @@ func (m *MetaDB) MarkChunkRefReady(ctx context.Context, chunk *chunkref.ChunkRef
 	return err
 }
 
-// InsertChunkRef inserts a new ChunkRef object to the datastore.
-func (m *MetaDB) InsertChunkRef(ctx context.Context, chunk *chunkref.ChunkRef) error {
```

Comment:
Removing `InsertChunkRef` breaks the collector test. `internal/app/collector/collector_test.go:126:29: collector.metaDB.InsertChunkRef undefined (type *metadb.MetaDB has no field or method InsertChunkRef)`

We should either add this back in or see if that test can take in `InsertChunkRefAsReady`

Reply:
I have modified the implementation so that the call to InsertChunkRef does not modify the status nor the timestamp of the object to be inserted in the function. I also renamed it to its original name.